src/backend/gl/src/command.rs

```diff
@@ -96,7 +96,12 @@ pub enum Command {
     /// Clear signed integer color drawbuffer of bound framebuffer.
     ClearBufferColorI(DrawBuffer, [i32; 4]),
     /// Clear depth-stencil drawbuffer of bound framebuffer.
-    ClearBufferDepthStencil(Option<pso::DepthValue>, Option<pso::StencilValue>),
+    ClearBufferDepthStencil {
+        depth: Option<pso::DepthValue>,
+        depth_mask: Option<bool>,
```

Comment:
do we really need to have it here? Could we instead have separate commands for setting the depth mask and stencil masks (i.e. `SetDepthMask`, `SetStencilMask`), so that the command encoder would insert them if necessary into the command stream?